src/navsat_transform.cpp

```diff
@@ -473,11 +488,12 @@ namespace RobotLocalization
     }
     else
     {
-      NavsatConversions::UTMtoLL(odom_as_cartesian.getOrigin().getY(),
-                                 odom_as_cartesian.getOrigin().getX(),
-                                 utm_zone_,
-                                 latitude,
-                                 longitude);
+      GeographicLib::UTMUPS::Reverse(utm_zone_,
```

Comment:
Same here. If we updated the methods in the other PR to call the GeographicLib equivalents, do we need to directly use GeographicLib here? Same for all the direct references to GeographicLib.